### ISSUE TITLE
[Pseudo Localization] Syntax tokens shouldn't be pseudo localized

### ISSUE THREAD

**Issue by nathanforce** [lingui/js-lingui#743]
**Describe the bug**

Pseudolocalization mutates syntax tokens, for example in the follow `select` syntax the key "screener" is mutated as well as the `select` keyword.

```
Error: Can't parse message. Please check correct syntax: "Ŷōũŕ ƥŕōvĩďēŕ śēńţ à {0, śēĺēćţ, śćŕēēńēŕ {ŵēĺĺńēśś àśśēśśmēńţ} other {{assessmentType}}} ƒōŕ ŷōũ ţō ţàķē. Ƥĺēàśē ţàķē à ƒēŵ mĩńũţēś ţō ćōmƥĺēţē ţĥĩś àśśēśśmēńţ ƀēƒōŕē ŷōũŕ ńēxţ àƥƥōĩńţmēńţ."
```

**To Reproduce**
Steps to reproduce the behavior, possibly with minimal code sample, e.g:

```
import { Trans } from "@lingui/react"

export default function App() {
   return (
    <div>
      Your provider sent a{' '}
            <Select
              value={isScreenerAssessment ? 'screener' : 'other'}
              screener="wellness assessment"
              other={assessmentType}
            />{' '}
            for you to take. Please take a few minutes to complete this
            assessment before your next appointment.
    </div>
}
```

**Expected behavior**
Syntax and tokens are unchanged, only actual language.

**Additional context**

- jsLingui version: 2.9.1

**Comment by tricoder42** [comment]
This definitely shouldn't happen, thanks for report.

Could you please checkout `next` branch and test, if the problem persists in v3? If so, feel free to send PR, but please use `next` branch as a base (it should be relesed early November). Let me know if I can help somehow

**Comment by MartinCerny-awin** [comment]
Hello @tricoder42 I can have a look at this issue

**Comment by tricoder42** [comment]
@MartinCerny-awin Perfect, thanks a lot!

**Comment by pauloborges** [comment]
Should this be really closed?

**Comment by semoal** [comment]
Probably this issue is already fixed mate.

**Comment by pauloborges** [comment]
Unfortunately no, the issue still happens with 3.4.0.

**Comment by semoal** [comment]
oh, alright i'll take a look then

**Comment by MartinCerny-awin** [comment]
I think Select is the last one that is missing. I am waiting for this issue to get fixed https://github.com/lingui/js-lingui/issues/919

@pauloborges is it Select that you are having issues with?

**Comment by pauloborges** [comment]
@MartinCerny-awin, exactly!

**Comment by semoal** [comment]
Okay i fix #919 asap

**Comment by semoal** [comment]
@MartinCerny-awin #919 is merged feel free to improve select pseudolocaliaztion.

You can introduce this test:
```js
  it("Select", () => {
    expect(
      pseudoLocalize(
        "{gender, select, male {He} female {She} other {<span>Other</span>}}"
      )
    ).toEqual(
      "{0, select, male {Ĥē} female {Śĥē} other {<span>Ōţĥēŕ</span>}}"
    )
  })
```

**Comment by pauloborges** [comment]
Awesome! Thanks for the quick fix!

A final question: any idea when a new release would be drafted?

**Comment by semoal** [comment]
> Awesome! Thanks for the quick fix!
> 
> A final question: any idea when a new release would be drafted?

We like to release new version each Tuesday. So the next Tuesday will include this fix =)

**Comment by semoal** [comment]
Released ^3.5.0

**Comment by pauloborges** [comment]
Thanks! Upgraded it and it's working as intended.
